LH0_W: 0.288959
LH0_X: -0.30153
LH0_Y: 0.639271
LH0_Z: 0.645689
LH0_x: 1.509119
LH0_y: -0.130085
LH0_z: -1.106622
LH1_W: -0.233641
LH1_X: 0.230571
LH1_Y: 0.6638
LH1_Z: 0.672026
LH1_x: 1.090903
LH1_y: -0.143942
LH1_z: -3.23766

freq: 100
unit: 100
